services/order.py

```diff
@@ -0,0 +1,35 @@
+from datetime import datetime
+
+from django.contrib.auth import get_user_model
+from django.db import transaction
+from django.db.models import QuerySet
+
+from db.models import Order, Ticket, MovieSession
+
+
+@transaction.atomic
+def create_order(
+        tickets: list[dict],
+        username: str,
+        date: datetime = None
+) -> None:
+    user = get_user_model().objects.get(username=username)
+    order = Order.objects.create(user=user)
+    if date:
+        order.created_at = date
+    order.save()
+    for ticket in tickets:
+        movie_session = MovieSession.objects.get(id=ticket["movie_session"])

```

Comment:
The `create_order` function should handle the case where a movie session with the given ID does not exist. Consider using `get_object_or_404` or handling the `MovieSession.DoesNotExist` exception to prevent potential errors.